data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/ResponseCardinality.java

```diff
@@ -0,0 +1,36 @@
+package org.opensearch.dataprepper.plugins.lambda.processor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ResponseCardinality {
+    STRICT("strict"),
+    AGGREGATE("aggregate");
+
+    private final String value;
+
+    ResponseCardinality(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
```

Comment:
```suggestion
    @JsonCreator
    public String getValue() {
```

This will allow this to work well with the schema generation as well.